Clinical-Genomics/scout
# Fix missing coverage report

This PR is bug fix #1380.

I did forget to include the code in the new sidebar! I'm using a link and javascript to submit the post request to remain consistent with the layout of the sidebar. Otherwise it would be of course easier with a submit button!

**How to test**:
1. The branch is already installed on scout stage
1. Go to whichever case and check that when you click on the sidebar, 'reports', you can see a 'Coverage report' option. When you click on the link the coverage report should open in a new page.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
